forms-flow-web/src/components/Dashboard/Dashboard.js

```diff
@@ -218,51 +218,49 @@ const Dashboard = React.memo(() => {
         >
           <Head items={headerList()} page="Metrics" />
           <div className="d-flex flex-wrap justify-content-between col-md-12">
-            <div className="input-group col-12 col-md-4 px-0">
-              <FormControl
-                type="search"
-                title="Search"
-                ref={searchInputBox}
-                onKeyPress={(e) => e.key === "Enter" && handleSearch()}
-                onChange={(e) => {
-                  setShowClearButton(e.target.value);
-                  setSearchTextInput(e.target.value);
-                  e.target.value === "" && handleSearch();
-                }}
-                autoComplete="off"
-                value={searchTextInput}
-                placeholder={t("Search...")}
-                style={{ backgroundColor: "#ffff" }}
-              />
-              {showClearButton && (
-                <InputGroup.Append
-                  onClick={() => onClear()}
-                  style={{ cursor: "pointer" }}
-                >
-                  <InputGroup.Text style={{ backgroundColor: "#ffff" }}>
-                    <i className="fa fa-times"></i>
-                  </InputGroup.Text>
-                </InputGroup.Append>
-              )}
+          <div className="custom-input-group col-12 col-md-4 px-0">
+      <InputGroup>
+        <FormControl
+          type="search"
+          title="Search"
+          ref={searchInputBox}
+          onKeyPress={(e) => e.key === "Enter" && handleSearch()}
+          onChange={(e) => {
+            setShowClearButton(e.target.value);
+            setSearchTextInput(e.target.value);
+            e.target.value === "" && handleSearch();
+          }}
+          autoComplete="off"
+          value={searchTextInput}
+          placeholder={t("Search...")}
+          className="bg-white"
+        />
+        {showClearButton && (
+          <InputGroup.Append className="d-flex cursor-pointer"onClick={() => onClear()}>
```

Comment:
the better way  to use this onClear function  is just passing to onClick variable else it will create an extra function